metrics/lib/common.bash

```diff
@@ -70,7 +70,7 @@ check_images()
 	local img req_images=( "$@" )
 	for img in "${req_images[@]}"; do
 		echo "ctr pull'ing: $img"
-		if ! ctr image pull "$img"; then
+		if ! sudo ctr image pull "$img"; then
```

Comment:
For uniformity, could you use $CTR_EXE instead of the command itself?

Reply:
@dborquez changes applied